app/monitoringapi.go

```diff
@@ -111,28 +113,22 @@ func startReadyChecker(ctx context.Context, tcpNode host.Host, eth2Cl eth2client
 	go func() {
 		ticker := clock.NewTicker(10 * time.Second)
 		epochTicker := clock.NewTicker(32 * 12 * time.Second) // 32 slots * 12 second slot time
-		previous := make(map[core.PubKey]bool)
-
-		// newCurrent returns a new current map, populated with all the pubkeys.
-		newCurrent := func() map[core.PubKey]bool {
-			current := make(map[core.PubKey]bool)
-			for _, pubkey := range pubkeys {
-				current[pubkey] = true
-			}
-
-			return current
+		currVAPICount := 0
```

Comment:
nit: this vapi count can be boolean too

Reply:
sure, but a count is more representative